node-placeholder/values.yaml

```diff
@@ -1,10 +1,11 @@
 replicaCount: 1
 
 image:
-  repository: us-central1-docker.pkg.dev/ucb-datahub-2018/core/node-placeholder-scaler
+  repository: 
+    us-central1-docker.pkg.dev/ucb-datahub-2018/core/node-placeholder-scaler
```

Comment:
This looks like a nested block rather than a `key: value` like in the previous version of the file. is chartpress being weird here?